lite/backends/xpu/target_wrapper.cc

```diff
@@ -77,6 +77,7 @@ XPUScratchPadGuard TargetWrapperXPU::MallocScratchPad(size_t size,
 
 std::string TargetWrapperXPU::multi_encoder_precision;  // NOLINT
 int TargetWrapperXPU::workspace_l3_size_per_thread{0};
+bool TargetWrapperXPU::xpu_conv_autotune{false};
```

Comment:
```suggestion
bool TargetWrapperXPU::conv_autotune{false};
```
TargetWrapperXPU下的变量，从命名上就不需要再有xpu了

Reply:
DONE